### ISSUE TITLE
snp_asGeneticPos step || I download the dataset, but it doesn't work in local computer?

### ISSUE THREAD

**Issue by fenfenge** [privefl/bigsnpr#393]
setwd('/home/genomes')
POS2 <- snp_asGeneticPos(CHR, POS,dir=paste0("chr", chr=c(1:22), ".OMNI.interpolated_genetic_map"),  ncores = 13)               

I saw the question that has been answered in github.
However, when I used the above code, it doesn't work and have some errors.

Error in { : task 1 failed - "lengths of 'url' and 'destfile' must match"

**Comment by privefl** [comment]
`dir` should be the directory where you have these files.

**Comment by fenfenge** [comment]
I used "setwd" set the working directory to dir. These files in the working directory. Thus, I don't write the director for dir.

Yesterday, I set dir='/home/genomes', it also doesn't work. And the error is Error in { : 
  task 1 failed - "cannot open URL 'https://github.com/joepickrell/1000-genomes-genetic-maps/raw/master/interpolated_OMNI/chr2.OMNI.interpolated_genetic_map.gz'".

**Comment by privefl** [comment]
You need to uncompress these, right?

**Comment by privefl** [comment]
Any update on this?

**Comment by zbjbiubiubiu** [comment]
Hi， 
   I also encountered the same problem. I downloaded the map file locally and decompressed it. The dir was set to dir=paste0("mydir/chr", chr=c(1:22), ".OMNI.interpolated_genetic_map"). Since I am using the hg38 data format, I have added rsid information. The running result reports an error Error in {: task 1 failed - "the condition has length > 1". How to solve it?
   POS2 <- snp_asGeneticPos(CHR, POS,dir=paste0("mydir/chr", chr=c(1:22), ".OMNI.interpolated_genetic_map"), rsid=SNP,ncores = 13)
